tests/config/SettingConfig.cc

```diff
@@ -0,0 +1,13 @@
+#define CATCH_CONFIG_MAIN
+#include "third-party/Catch/single_include/catch.hpp"
+
+#include "config/SettingConfig.h"
+
+TEST_CASE("Test SettingConfig", "[config]")
+{
+    emp::SettingConfig config_set;
+    config_set.AddSetting<int>("test_non_combo") = {5};  
```

Comment:
does the 5 have to be in {}’s ... ? I might just not quite be understanding what’s going on here

Reply:
It does not have to be in {}s, but they also don't hurt anything

Reply:
Ah, I had assumed they were there b/c they had to be there for some obscure reason. Might be worth considering removing just to make the code easier to read the next time someone digs thru it (and they don’t assume there’s an obscure reason they have to be there)